src/utils/feeBreakdown.ts

```diff
@@ -122,7 +125,11 @@ export const createPassportSchema = () => {
     .object({
       "application.fee.calculated": feeSchema.optional().default(0),
       "application.fee.payable": feeSchema,
-      "application.fee.payable.includesVAT": booleanSchema,
+      "application.fee.serviceCharge": feeSchema.optional().default(0),
+      "application.fee.serviceCharge.VAT": feeSchema.optional().default(0),
+      "application.fee.fastTrack": feeSchema.optional().default(0),
+      "application.fee.fastTrack.VAT": feeSchema.optional().default(0),
+      // `${z.string()}.VAT`: feeSchema.optional().default(0),
```

Comment:
Similar to above - expect to be able to specify a dynamic `.VAT` property here but haven't landed on correct syntax yet !